src/Communication/SAF.Communication.PubSub.Cde/ISubscriptionInternal.cs

```diff
@@ -10,6 +10,6 @@ namespace SAF.Communication.PubSub.Cde
 {
     internal interface ISubscriptionInternal : ISubscription
     {
-        void With(Action<string, TheProcessMessage> callback);
+        void SetRawHandler(Action<string, TheProcessMessage> callback);
```

Comment:
Is it possible to avoid the `SetRawHandler` call completely?

Reply:
When we pass an action into the constructor we cannot decide whether its the Handler or the RawHandler.